router/auth.go

```diff
@@ -84,6 +84,42 @@ func authPassThrough(log *log.Logger, w http.ResponseWriter, req *http.Request,
 
 func authInjectECS(log *log.Logger, w http.ResponseWriter, req *http.Request, target *targetPassThroughAuth) bool {
 	req.SetBasicAuth(target.config.Username, target.config.Password)
+	var didWhat string
+	var toWhat string
+	actionUrl := strings.Split(req.URL.String(), "/") // Example: "ecs/ACCESS/FWVERSION/" or "ecs/sam/ForceSim1/"
+	if len(actionUrl) == 5 {
+		switch actionUrl[2] {
+		case "ACCESS":
```

Comment:
Let's cater for all known API calls (in energy-defs.ts) and don't allow the pass-through to succeed for any other calls